exp-models/addon/authenticators/jam-osf-jwt.js

```diff
@@ -34,6 +48,14 @@ export default Base.extend({
         return this._post(accessToken).then(function(res) {
             res.data.attributes.accessToken = accessToken;
             return res.data.attributes;
+        })
+            .then((res) => {
+               // Only allow someone to login to the admin panel if they have have namespace-level permissions for this project
+                return this._askNamespace(res.token)
+                    .then(() => {return res;})
```

Comment:
You don't need the `{}` or the `return` here.
`.then(() => res)` will do the same thing